Description: Remove -pedantic option from g++
 This patch removes the -pedantic option from g++ to prevent uncritical compile
 errors.
Author: J.H.M. Dassen (Ray) <jdassen@debian.org>

--- pstoedit-3.50.orig/configure.ac
+++ pstoedit-3.50/configure.ac
@@ -405,7 +405,7 @@ if test "$GXX" = yes; then
   else
     AC_MSG_RESULT([GNU])
 #   need to disable long long warning here because of ImageMagick using it
-    SPECIAL_CXXFLAGS="-DBUGGYGPP -pedantic -Wall -Wwrite-strings -Wcast-qual -Wpointer-arith -Wno-long-long "
+    SPECIAL_CXXFLAGS="-DBUGGYGPP -Wall -Wwrite-strings -Wcast-qual -Wpointer-arith -Wno-long-long "
   fi
   CXXFLAGS="-g" 
 elif test `echo "$CXX" | egrep -c 'xlC$'` -eq 1; then

